src/app/compare/CompareButtons.m.scss

```diff
@@ -0,0 +1,11 @@
+.svgIcon {
+  height: 1.3em;
+  margin: 2.5px;
+}
+
+.inlineModSlotIcon {
+  display: inline-block;
+  vertical-align: middle;
+  height: 1.3em;
+  width: 1.3em;
```

Comment:
I'll just go on the record as saying I generally prefer `px` to `em`.

Reply:
i figured this would lock it to its accompanying text size without having to edit both if text changes. hope i am not misinterpreting what it'll do

Reply:
Yeah it will. But since it's a multiplier on the base font size you can end up with weird non-integer image sizes.